.goreleaser.yml

```diff
@@ -64,71 +64,156 @@ release:
   # (we want Releaser to be responsible for doing all the GitHub release manipulations)
 
 dockers:
+  #
   # The following templates are for the Alpine-based image.
+  #
+
   # i386
   - image_templates:
     - "launchdarkly/ld-relay:{{ .Version }}-i386"
-    - "launchdarkly/ld-relay:v{{ .Major }}-i386"
-    - "launchdarkly/ld-relay:latest-i386"
     use: buildx
     goos: linux
-    goarch: '386'
+    goarch: "386"
     dockerfile: Dockerfile.goreleaser
     skip_push: false
     build_flag_templates:
-    - "--pull"
-    - "--platform=linux/386"
+      - "--pull"
+      - "--platform=linux/386"
+
+  - image_templates:
+    - "launchdarkly/ld-relay:v{{ .Major }}-i386"
+    use: buildx
+    goos: linux
+    goarch: "386"
+    dockerfile: Dockerfile.goreleaser
+    skip_push: '{{ eq (env "LD_RELEASER_UPDATE_MAJOR") "true" }}'
```

Comment:
It might be my misunderstanding of skip_push but shouldn't this checking if LD_RELEASER_UPDATE_MAJOR is false?